src/com/hughes/android/dictionary/DictionaryApplication.java

```diff
@@ -313,6 +314,12 @@ public Theme getSelectedTheme() {
         final String theme = prefs.getString(appContext.getString(R.string.themeKey), "themeLight");
         if (theme.equals("themeLight")) {
             return Theme.LIGHT;
+        } else if (theme.equals("themeSystem")) {
+            int mode = (appContext.getResources().getConfiguration().uiMode &
+                        Configuration.UI_MODE_NIGHT_MASK);
+            return ((mode == Configuration.UI_MODE_NIGHT_YES) ?
+                    Theme.DEFAULT :
```

Comment:
Is it intentional that for UI_MODE_NIGHT_UNKNOWN it would default to the LIGHT theme?

Reply:
That reflects my biases. Since dark mode is the default, it would be better if it uses dark mode in the unknown state. I can change this too.